### PR TITLE
(MAINT) README updates!

### PR DESCRIPTION
**Pull request by mckern**
We want to cut a new release, and that means updating the documentation. In addition to adding doc for the `inspect` command and the `RETRY_COUNT` environment variables, this PR includes a couple of formatting changes and some revised descriptions for common Vanagon activities. Please view it as a rendered document and make sure that nothing looks funky.

giphy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,28 +101,35 @@ Increase verbosity of output.
...
 vmpooler to access the API. Without this token, the default lifetime of vms
 will be much shorter.
 
-##### LOCK\_MANAGER\_HOST
+##### `LOCK_MANAGER_HOST`
 The name of the host where redis is running. Redis is used to handle a lock
 when using the hardware engine. It defaults to *redis*, with no domain.
 
-#### LOCK\_MANAGER\_PORT
+#### `LOCK_MANAGER_PORT`
```
Comment (McdonaldSeanp) [inline README.md]: should be 5 #'s

File: README.md [hunk 2]
```diff
@@ -100,28 +101,35 @@ Increase verbosity of output.
...
 Port of the system where redis is running. Defaults to *6379*.
 
+#### `RETRY_COUNT`
```
Comment (McdonaldSeanp) [inline README.md]: again, 5 #'s

File: README.md (hunk 1 above)
Comment (mckern) [inline README.md]: I'll fix that up but do note I didn't bring that to the party

Comment (McdonaldSeanp) [inline README.md]: 👍 it just looked weird, thanks for fixing it

**Comment by mckern** [comment]
@McdonaldSeanp fixed up

**Comment by McdonaldSeanp** [comment]
Coo'

**Comment by McdonaldSeanp** [comment]
👍

**Pull request merged**